notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2261 @@
+{
```

Comment:
<p>"how to use <strong>the</strong> Vertex AI regression model evaluation component to evaluate <strong>a</strong> custom tabular regression model." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>